opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -0,0 +1,217 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import threading
+from enum import Enum
+
+from opentelemetry.semconv.trace import SpanAttributes
+
+# TODO: will come through semconv package once updated
+_SPAN_ATTRIBUTES_ERROR_TYPE = "error.type"
+_SPAN_ATTRIBUTES_NETWORK_PEER_ADDRESS = "network.peer.address"
+_SPAN_ATTRIBUTES_NETWORK_PEER_PORT = "network.peer.port"
+_METRIC_ATTRIBUTES_CLIENT_DURATION_NAME = "http.client.request.duration"
+
+_client_duration_attrs_old = [
+    SpanAttributes.HTTP_STATUS_CODE,
+    SpanAttributes.HTTP_HOST,
+    SpanAttributes.NET_PEER_PORT,
+    SpanAttributes.NET_PEER_NAME,
+    SpanAttributes.HTTP_METHOD,
+    SpanAttributes.HTTP_FLAVOR,
+    SpanAttributes.HTTP_SCHEME,
+]
+
+_client_duration_attrs_new = [
+    _SPAN_ATTRIBUTES_ERROR_TYPE,
+    SpanAttributes.HTTP_REQUEST_METHOD,
+    SpanAttributes.HTTP_RESPONSE_STATUS_CODE,
+    SpanAttributes.NETWORK_PROTOCOL_VERSION,
+    SpanAttributes.SERVER_ADDRESS,
+    SpanAttributes.SERVER_PORT,
+    # TODO: Support opt-in for scheme in new semconv
+    # SpanAttributes.URL_SCHEME,
```

Comment:
Shouldn't we just create our own constant in the meantime?

Reply:
The TODO refers to the feature for opting into enabling these attributes, not that the attributes don't exist yet.